packages/eas-cli/schema/metadata-0.json

```diff
@@ -472,49 +472,74 @@
         "description": "Info provided to the App Store review team to help them understand how to use your app.",
         "required": [
           "firstName",
-          "lastName"
+          "lastName",
+          "email",
+          "phone"
         ],
         "defaultSnippets": [
           {
+            "label": "Basic",
+            "description": "Only define the contact details in case communcation is needed with the App Store review team.",
+            "body": {
+              "firstName": "$1",
+              "lastName": "$2",
+              "email": "$3",
+              "phone": "+$4"
+            }
+          },
+          {
+            "label": "Full",
+            "description": "Define the demo account credentials, additional info, and contact details for the App Store review team.",
             "body": {
               "firstName": "$1",
-              "lastName": "$2"
+              "lastName": "$2",
+              "email": "$3",
+              "phone": "+$4",
+              "demoUsername": "$5",
+              "demoPassword": "$6",
+              "demoRequired": true,
+              "notes": "$7"
             }
           }
         ],
         "properties": {
-          "demoUsername": {
+          "firstName": {
             "type": "string",
-            "description": "Optional username for testing the app."
+            "description": "First name of contact in case communication is needed with the App Store review team.",
+            "minLength": 1
           },
-          "demoPassword": {
+          "lastName": {
             "type": "string",
-            "description": "Optional password for testing the app."
+            "description": "Last name of contact in case communication is needed with the App Store review team.",
+            "minLength": 1
           },
-          "attachment": {
+          "email": {
             "type": "string",
-            "description": "Local file path to upload to the review team."
+            "description": "Email address of contact in case communication is needed with the App Store review team.",
+            "format": "email"
           },
-          "firstName": {
+          "phone": {
             "type": "string",
-            "description": "Developer first name for the reviewer to contact."
+            "description": "Phone number of contact in case communication is needed with the App Store review team. Preface the phone number with ‘+’ followed by the country code (for example, +44 844 209 0611)"
           },
```

Comment:
It's probably not the best to add phone number validation with regex in this schema. That's why I omitted it.